### ISSUE TITLE
SA1516 (ElementsMustBeSeparatedByBlankLine) reported for adjacent fields; code fix does not work

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#548]
When a field follows a multi-line field, SA1516 gets reported. In this case, the code fix is proposed but the set of changes is empty. The following code demonstrates this case (SA1516 is reported for `experiment2`).

```csharp
private string experiment =
    string.Empty;
private string experiment2;
```

* If SA1516 should be reported in this case, the code fix should be updated to handle it.
* Otherwise, SA1516 should not be reported here.
